README.md

```diff
@@ -2,33 +2,33 @@
 
 # Sniper
 
-> An easy to run simple bot that lets you snipe messages in your Discord server.
+> An easy to run simple bot that lets you snipe and editsnipe messages in your Discord server.
 
 ## Setup
 
 Node.js 16.6.0 or newer is required.
 
 1. Run:
 
-```
-$ git clone https://github.com/DankMemer/sniper.git
+```bash
+$ git clone https://github.com/DankMemer/sniper.git 
 $ cd ./sniper
 ```
 
 2. Create config.json:
 
-```
+```json
 {
-	"token": "<Your bot's token>",
-	"application_id": "<Your application's id>"
+	"token": "<Your bot's token>", //obtain from https://discord.com/developers/applications
```

Comment:
I don't think it's right place for this comments, especially with github highlighting.